BH/Modules/Item/ItemDisplay.cpp

```diff
@@ -404,6 +404,310 @@ enum Operation
 	NONE
 };
 
+enum FilterCondition
+{
+	COND_AND,
+	COND_OR,
+	COND_ETH,
+	COND_SOCK,
+	COND_SET,
+	COND_MAG,
+	COND_RARE,
+	COND_UNI,
+	COND_AMAZON,
+	COND_SORCERESS,
+	COND_NECROMANCER,
+	COND_PALADIN,
+	COND_BARBARIAN,
+	COND_DRUID,
+	COND_ASSASSIN,
+	COND_CRAFTALVL,
+	COND_PREFIX,
+	COND_SUFFIX,
+	COND_AUTOMOD,
+	COND_MAPID,
+	COND_MAPTIER,
+	COND_CRAFT,
+	COND_RW,
+	COND_NMAG,
+	COND_SUP,
+	COND_INF,
+	COND_NORM,
+	COND_EXC,
+	COND_ELT,
+	COND_ID,
+	COND_ILVL,
+	COND_QLVL,
+	COND_ALVL,
+	COND_CLVL,
+	COND_FILTLVL,
+	COND_DIFF,
+	COND_RUNE,
+	COND_GOLD,
+	COND_GEMMED,
+	COND_GEMTYPE,
+	COND_GEM,
+	COND_ED,
+	COND_DEF,
+	COND_MAXDUR,
+	COND_RES,
+	COND_FRES,
+	COND_CRES,
+	COND_LRES,
+	COND_PRES,
+	COND_IAS,
+	COND_FCR,
+	COND_FHR,
+	COND_FBR,
+	COND_LIFE,
+	COND_MANA,
+	COND_QTY,
+	COND_GOODSK,
+	COND_GOODTBSK,
+	COND_FOOLS,
+	COND_LVLREQ,
+	COND_ARPER,
+	COND_MFIND,
+	COND_GFIND,
+	COND_STR,
+	COND_DEX,
+	COND_FRW,
+	COND_MINDMG,
+	COND_MAXDMG,
+	COND_AR,
+	COND_DTM,
+	COND_MAEK,
+	COND_REPLIFE,
+	COND_REPQUANT,
+	COND_REPAIR,
+	COND_ARMOR,
+	COND_BELT,
+	COND_CHEST,
+	COND_HELM,
+	COND_SHIELD,
+	COND_GLOVES,
+	COND_BOOTS,
+	COND_CIRC,
+	COND_DRU,
+	COND_BAR,
+	COND_DIN,
+	COND_NEC,
+	COND_SIN,
+	COND_SOR,
+	COND_ZON,
+	COND_SHOP,
+	COND_EQUIPPED,
+	COND_1H,
+	COND_2H,
+	COND_AXE,
+	COND_MACE,
+	COND_SWORD,
+	COND_DAGGER,
+	COND_THROWING,
+	COND_JAV,
+	COND_SPEAR,
+	COND_POLEARM,
+	COND_BOW,
+	COND_XBOW,
+	COND_STAFF,
+	COND_WAND,
+	COND_SCEPTER,
+	COND_EQ1,

```

Comment:
EQ1-7, WP1-13, and CL1-7 all have more specifically named counterparts (HELM, SWORD, etc). So we can removed them here, remove their cases, and then just change their value in `condition_map `
https://wiki.projectdiablo2.com/wiki/Item_Filtering#Armor_Groups

Reply:
Yeah I noticed that, I left it in there in case someone was already using them. We can remove them if we can inform loot filter creators that will no longer work

Reply:
So what I'm saying is we _do_ keep them, but we remove their cases since they're identical to their more properly named counterparts. So EQ1 would go from `{"EQ1", COND_EQ1}` to `{"EQ1", COND_HELM}`. And then because they're the same, we can remove the EQ/WP/CL items from the enum and also remove their cases in the switch. We're left with the same functionality as before but with less bloat

Reply:
I see what you mean, yeah you're right I can do that.